### ISSUE TITLE
[US-6-4][Low] Locator selecting through the locators list doesn't mark the locator on the desktop

### ISSUE THREAD

**Issue by ioffem** [jdi-testing/jdn-ai#271]
JDN v 3.1.157
Back-end v 2022-03-30-06.47.33
Chrome

1) open any page
2) open JDN
3) identify
4) select locator in the locators list

AR: locator is not marked on the desktop
ER: locators should be marked on the desktop if it was selected in the locators list

**Comment by AlexeyGirin** [comment]
@IoffeM - перепроверить

**Comment by IoffeM** [comment]
found out that the issue is in logic of locators marking only after page scrolling. it's a known issue, but not high priority

**Comment by IoffeM** [comment]
reopened 

JDN v 3.1.125
Back-end v 2022-02-12-11.36.19
Chrome
https://cbracco.github.io/html5-test-page/

is still reproducible
![mark](https://user-images.githubusercontent.com/98175427/154681079-ac07a752-61b1-4491-94e5-7c2adb52c0ed.gif)

**Comment by MariiaNebesnova** [comment]
воспроизводится в случае, когда до элемента подствеки еще не скроллили и он еще не отрисован на странице

**Comment by IoffeM** [comment]
ok, it's a low priority, let's fix it later

**Comment by MariiaNebesnova** [comment]
3.1.179
